web/src/main/resources/assets/js/graphhopper/GHRequest.js

```diff
@@ -232,6 +259,10 @@ GHRequest.prototype.doRequest = function (url, callback) {
     $.ajax({
         timeout: 30000,
         url: url,
+        beforeSend: function(request) {
+            // todonow: maybe rename
+            request.setRequestHeader("gh-client", "web-ui")
```

Comment:
this can be used to recognize that a request was sent using this client

Reply:
> Is there a way we could include some version?

Sure we could add another header with a version. Would be nice to get this via maven or even git commit hash or tag during from the build process? Or simply hard-code it ;)

> And isn't it usually called "User-Agent"

Yeah I think user-agent is used for browser version, using a custom / non-standard header for what we want to achieve here seems reasonable to me

Reply:
> Would be nice to get this via maven or even git commit hash or tag during from the build process?

Ah, yes. This sounds doable but for a later issue ;)

Reply:
I would not recommend overwriting the User-Agent.

IIRC a lot of browsers don't allow that, also some CDNs/Firewalls might check that string and if it looks tempered they might assume something funny is going on and potentially block the request.

Reply:
> Ah, yes. This sounds doable but for a later issue ;)

We could still hard-code it for now, that would be more information than not sending any version (even though it might be outdated sometimes) :)

Reply:
You could take a look at the buildnumber maven plugin: https://www.mojohaus.org/buildnumber-maven-plugin/usage.html